src/main/java/org/sunbird/storage/service/StorageServiceImpl.java

```diff
@@ -55,6 +55,25 @@ public SBApiResponse uploadFile(MultipartFile mFile, String containerName) {
 			FileOutputStream fos = new FileOutputStream(file);
 			fos.write(mFile.getBytes());
 			fos.close();
+			return uploadFile(file, containerName);
+		} catch (Exception e) {
+			logger.error("Failed to upload file. Exception: ", e);
+			response.getParams().setStatus(Constants.FAILED);
+			response.getParams().setErrmsg("Failed to upload file. Exception: " + e.getMessage());
+			response.setResponseCode(HttpStatus.INTERNAL_SERVER_ERROR);
+			return response;
+		} finally {
+			if (file != null) {
+				file.delete();
+			}
+		}
+	}
+
+	@Override
+	public SBApiResponse uploadFile(File file, String containerName) {
+		SBApiResponse response = new SBApiResponse();
+		response.setId(Constants.API_FILE_UPLOAD);
```

Comment:
You need to use ProjectUtil.createDefaultResponse instead of directly creating object.